### DISCUSSION TITLE
[Projects Beta] Display a task progress bar for issues

### DISCUSSION THREAD

**Discussion by MarkBennett** [community/community#10327] [Projects and Issues]
In the issue list on a repo, it's very handy to see that green bar which shows progress on subtasks for an issue so I can see at a glance how work is progressing.

It would be nice if issues on the new Projects Beta showed subtask progress bars too.

**Comment by andrewhsu** [comment]
is this suggestion a duplicate of https://github.com/github/feedback/discussions/7575 ?

**Reply by 5hee75** [reply]
I'd call it related, but not exactly a duplicate. That other issue is about listing the tasks it seems, but this is more about showing the high level progress of tasks in terms of a percentage or x/n completed.
